src/algorithms/cDIIS.jl

```diff
@@ -116,14 +131,17 @@ end
 function diis_build_matrix(n_diis_size::Int, m::Int, state::cDIISstate)
     A = zeros(m +1,m +1)
 
-    # Since the Matrix A is Hermitian, we only have to calculate
-    # the upper triagonal and can use the Julia function 'Hermitian'
+    # Since the Matrix A is symmetric, we only have to calculate
+    # the upper triagonal and can use the Julia object 'Symmetric'
     # to fill the lower triagonal of the matrix. This also allowes
-    # Julia to use optimized algorithems for Hermitian matrices.
+    # Julia to use optimized algorithems for symmetric matrices.
+
+    # TODO Draw the shape of b with the overlaps and indices as I do it
```

Comment:
I have looked into your implementation and could not find differences in conventions used. You store the error overlaps in an array of arrays which has the downside, that you cannot map a whole row to an existing array as I do it in this implementation. In addition to that you store the values of the symmetric matrix twice. I'm pretty sure Julia does not store the values twice in a symmetric matrix and this was probably not what you wanted me to change. Can you clarify what exactly you would like to be changed?